src/aap_eda/tasks/orchestrator.py

```diff
@@ -99,6 +104,8 @@ def _manage(process_parent_type: str, id: int) -> None:
         ActivationStatus.RUNNING,
         ActivationStatus.WORKERS_OFFLINE,
     ]:
+        assign_request_id(str(uuid.uuid4()))
```

Comment:
I think this is wrong. At this point the request record should have a request id and we should use that one, no a new one. Am I wrong?

Reply:
Fixed